Corona-Warn-App/src/main/res/layout/tracing_content_low_view.xml

```diff
@@ -74,7 +74,7 @@
 
         <androidx.constraintlayout.widget.ConstraintLayout
             android:id="@+id/row_saved_days"
-            gone="@{state.getRiskContactLast(context) != null}"
+            visible="@{state.getRiskContactLast(context) == null || state.inDetailsMode}"
```

Comment:
Why not use `gone` instead. No point in maintaining both adapters IMHO

Reply:
I implemented it with `gone` in the beginning and it looked like this: 
`gone="@{state.getRiskContactLast(context) != null &amp;&amp; !state.inDetailsMode}"`
which is really hard to understand (`&amp;&amp;` is the logical AND operator (&&) in data binding )

The version here with `visible` is much easier to understand. 

I don't see any "costs" of maintaining this additional adapter, but I see some value in simplifying the expression.

Reply:
How about moving the logic to the state object --> `gone=@{state.isGoneOnContentLowView(context)}` for  instance

Reply:
Good idea. Moved the logic to the state object and wrote tests for it.